stock_picking_batch_extended/tests/test_batch.py

```diff
@@ -46,19 +54,17 @@ def create_simple_picking(cls, product_ids, batch_id=False):
         return cls.picking_model.with_context(planned=True).create(
             {
                 "picking_type_id": cls.picking_type_out.id,
-                "location_id": cls.stock_loc.id,
-                "location_dest_id": cls.customer_loc.id,
+                "location_id": cls.stock_location.id,
+                "location_dest_id": cls.customer_location.id,
                 "batch_id": batch_id,
                 "move_ids": [
-                    (
-                        0,
-                        0,
+                    Command.create(
                         {
                             "name": "Test move",
                             "product_id": product_id,
-                            "product_uom_qty": 1,
-                            "location_id": cls.stock_loc.id,
-                            "location_dest_id": cls.customer_loc.id,
+                            "product_uom_qty": 10,
```

Comment:
I propose to increase the quantity to easily simulate partial and backorder usage behavior